controllers/userController.js

```diff
@@ -38,11 +38,11 @@ router.get('/profile/:id', (req, res) => {
     }).then((data) => {
         // console.log("this is data", data);
         console.log("this is data", data.zipcode);
-        res.send({
+        res.render('profile', {
```

Comment:
`res.render()` is an express-handlebars function that allows you to tell it what template to render, and pass any data you need to pass to it.  

The template needed to be in the `views` folder, not `partials` (partials is for pieces of html that you want to render in a template, which means you'll still need to have a template in views with the name you pass to `res.render()`, and you'll have to reference the partial in that template if you want to use it)

I also added the zipcode to the page just to show you how to use data that is passed to a handlebars template.